### ISSUE TITLE
More detail about base_url and auth_endpoint, and subdirectories

### ISSUE THREAD

**Issue by atomcorp** [tylergaw/netlify-cms-github-oauth-provider-server-example#1]
Hey! Your blog post and repos have been so helpful for me getting this set up. Thanks so much for documenting your process!

tl:dr - it would be great to add a little more info about: 

- base_url - this is strictly the root domain, nothing more
- auth_endpoint - if the server lives in a subdirectly, this should lead to the "auth" endpoint, excluding the base_url

So, like I said great resource however I did get stuck on one bit, which was outside the scope what you needed to do, but I think is a likely scenario for others following along.

All you server code is at the root domain, eg. https://netlify-cms-github-oauth-provider-example.glitch.me, therefore everything you describe works. 

However if the server lives in a subdirectory things work up to a point, then confusingly don't work without warning. Basically it gets to the callback, authenticates with GitHub successfully and then nothing happens, the window doesn't close, NetlifyCMS isn't told about the authentication.

The key is the base_url must match the domain only, not the whole path to the server - it's necessary for the postMessage stuff. Then to direct NetlifyCMS to the actual auth point, you need to add auth_endpoint.

For example: my server endpoints lived at my-site-vercel.com/api/ (so my-site-vercel.com/api/auth, my-site-vercel.com/api/callback). Therefore i needed to do:

```
base_url: my-site-vercel.com
auth_endpoint: api/auth
```

There are some other topics that helped me figure this out, ([this comment](https://github.com/netlify/netlify-cms/issues/1285#issuecomment-451058395)) but it's a little spread out and confusing. I think it would be good to have it with your article, which is the clearest resource about all this I found.

Cheers!
